src/ui/components/UserAvatar/index.js

```diff
@@ -14,9 +14,14 @@ type Props = {|
 const UserAvatar = ({ className, user }: Props) => {
   const _className = makeClassName('UserAvatar', className);
 
-  return (user && user.picture_type && user.picture_type.length ?
-    <img alt="" className={_className} src={user.picture_url} /> :
-    <Icon className={_className} name="anonymous-user" />
+  return (
+    <div className={_className}>
+      {
+        (user && user.picture_type && user.picture_type.length) ?
+          <img alt="" src={user.picture_url} /> :
```

Comment:
I would add a className to target directly above. `UserAvatar-image` would be fine I think.